src/sessions.ts

```diff
@@ -194,7 +195,7 @@ export class ClientSession extends TypedEventEmitter<ClientSessionEvents> {
 
     this.operationTime = undefined;
     this.owner = options.owner;
-    this.defaultTransactionOptions = Object.assign({}, options.defaultTransactionOptions);
```

Comment:
This isn't directly related to the code changes, but is nice to have and helps us have less `Object.assign` calls in our code